### DISCUSSION TITLE
Add vim bindings to navigate tasks

### DISCUSSION THREAD

**Discussion by dcodesdev** [vercel/turborepo#8313] [Ideas]
### Goals

Most developers are used to the vim bindings to move up and down with `k` and `j`

It would be nice to be able to move up and down the tasks lists in the terminal with `k` and `j` as well.

![image](https://github.com/vercel/turbo/assets/101001810/bed2ed8f-d1fc-4944-9f21-b8142d59fd00)

### Background

Most users are used to VIM so they might assume they can also move up and down with the same bindings.

### Proposal

Easy to implement! just a few code changes, I can do it.

**Comment by anthonyshew** [comment]
While Vim is great, we don't want to add more binds to this interface. If we did, we would have to start iterating over the various editors binds, competing with the binds of other folks, etc. The goal here isn't to build a full multiplexer, so we want to avoid that sort of scope expansion.

**Reply by anthonyshew** [reply]
We've now changed our tune about this! Merged in https://github.com/vercel/turborepo/pull/9243.
